scene/resources/primitive_meshes.cpp

```diff
@@ -361,8 +361,8 @@ void CapsuleMesh::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("set_rings", "rings"), &CapsuleMesh::set_rings);
 	ClassDB::bind_method(D_METHOD("get_rings"), &CapsuleMesh::get_rings);
 
-	ADD_PROPERTY(PropertyInfo(Variant::REAL, "radius", PROPERTY_HINT_RANGE, "0.1,100.0,0.1"), "set_radius", "get_radius");
-	ADD_PROPERTY(PropertyInfo(Variant::REAL, "mid_height", PROPERTY_HINT_RANGE, "0.1,100.0,0.1"), "set_mid_height", "get_mid_height");
+	ADD_PROPERTY(PropertyInfo(Variant::REAL, "radius", PROPERTY_HINT_RANGE, "0.001,100.0,0.001"), "set_radius", "get_radius");
```

Comment:
I think it's fine to support down to 0.001, but using 0.001 as increment is overkill IMO. I'd keep 0.1 as increment, if people need a finer value, they can set it manually.

Reply:
I have thinked same as you, 0.001 for precision but 0.1 for increment, but after if in editor you enter 0.05 the UI change to 0.1, it's confusing no ?

Reply:
Height of a sphere depends on the shape radius. (height = radius * 2)
Sphere is not an ellipse. When I set the radius, the shape turns into elipse. I think the height value should be removed.

Reply:
That's CapsuleMesh, it's not meant to be a sphere, but a capsule :)

Reply:
A message was removed ? I don't understand...

Reply:
Yes, I was answering to:

> Height of a sphere depends on the shape radius. (height = radius * 2)
Sphere is not an ellipse. When I set the radius, the shape turns into elipse. I think the height value should be removed.

Reply:
Ho ok.

I let this PR as is or we must think a way to have 0.1 increment and a manual input (and display) to 0.001 precision. I guess this is not supported by Godot for now.

Reply:
No it's fine as is. I'll just wait for 3.1 even though it's trivial, as otherwise people keep bombarding with enhancements while we're in release freeze :)